[K9VULN-4229] Add troubleshooting note for Datadog-hosted SCA scans (#28224)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

There are a series of steps that must run successfully for vulnerability information to appear either in the [Software Catalog][16] **Security** view or in the [Vulnerabilities explorer][12]. It is important to check each step when investigating this issue.

> ## No results with Datadog-hosted scans for a repository using `git-lfs`

Reviewer: ```suggestion
## No results from Datadog-hosted scans for a repository using `git-lfs`
```

> Software Composition Analysis Datadog-hosted scans do not support repositories that use Git Large File Storage (`git-lfs`). If your repository uses `git-lfs`, you can [setup a CI pipeline][18] to run an analysis and upload the results to Datadog instead.

Reviewer: ```suggestion
Datadog-hosted scanning for Software Composition Analysis (SCA) does not support repositories that use Git Large File Storage (`git-lfs`). If your repository uses `git-lfs`, [set up the analysis in a CI pipeline][18] and upload the results to Datadog instead.
```
